eradiate/radprops/rad_profile.py

```diff
@@ -39,6 +39,7 @@
 from eradiate import __version__
 from .absorption import compute_sigma_a
 from .rayleigh import compute_sigma_s_air
+from ..data.absorption_spectra import _AbsorptionGetter
```

Comment:
I assume that's not very clean, however I do not see another way to import the list of paths to the absorption datasets.